### PR TITLE
add mlwc package v.0.1.0

### PR DESCRIPTION
**Pull request by ToAmano**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ToAmano** [comment]
@conda-forge/help-python

ready for review! Thank you!

**Comment by ToAmano** [comment]
@conda-forge/staged-recipes

ready for review! Thank you!

**Review by xhochy — CHANGES_REQUESTED** [review]
Please remove all commented lines and only use packages from conda-forge.

**Comment by ToAmano** [comment]
@xhochy 
Thank you for reviewing. I've removed all the comments and packages except conda-forge. Please check it again.

**Comment by xhochy** [comment]
If I look at the sources, it looks like this is a pure Python package, right? Then it should also be `noarch: python`.

Please also fix the inclusion of the license in the source tarball. I see you are the main author of the package, so it's better to fix it upstream instead of bundling the license with the recipe.

**Comment by ToAmano** [comment]
@xhochy
Is it necessary to pass the test on `win_64`? It seems Pytorch can not be installed on win_64. 
I would appreciate any suggestion. Thank you.

File: recipes/MLWC/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --prefix=$PREFIX
+
+requirements:
+  host:
+    - cmake
```
Comment (xhochy) [inline recipes/MLWC/meta.yaml:21]: This and boost below are typically only required foe compiled packages. Can you simply remove them?

File: recipes/MLWC/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    - pytorch
+    - torchinfo
+    - ase ==3.23.0
+    - rdkit >=2023.03.0    
+    - mdtraj
+    - pyyaml
+    - statsmodels
+    - matplotlib-base
+
+test:
```
Comment (xhochy) [inline recipes/MLWC/meta.yaml:40]:
```suggestion
test:
  requires:
    - pip
```

File: recipes/MLWC/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  imports:
+    - torch 
+    - rdkit
+    - ase
+  commands:
```
Comment (xhochy) [inline recipes/MLWC/meta.yaml:45]:
```suggestion
  commands:
    - pip check
```

**Review by xhochy — CHANGES_REQUESTED** [review]
It's OK for Windows to error on a missing pytorch dependency if this is noarch

File: recipes/MLWC/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+test:
+  requires:
+    - pip
+  requires:
+    - pip
+  imports:
+    - torch 
+    - rdkit
+    - ase
```
Comment (xhochy) [inline recipes/MLWC/meta.yaml:46]: This should be removed
```suggestion
```

**Review by xhochy — CHANGES_REQUESTED** [review]

**Comment by ToAmano** [comment]
@xhochy 

Ready for review. Thank you.

**Review by xhochy — APPROVED** [review]

**Pull request merged**